.prospector.yaml

```diff
@@ -0,0 +1,47 @@
+inherits:
+  - strictness_medium
+
+pylint:
+  options:
+    bad-names: foo,baz,toto,tutu,tata
+    # max-args default = 5
+    max-args: 7
+    # max-locals default = 15
+    max-locals: 16
+    # max-branches default = 15
+    max-branches: 15
+  disable:
+    - line-too-long
+    - protected-access
+    - unused-import
```

Comment:
I removed them because there were a lot of error messages raised by them. 
I meant to turn them back off or add a comment suggesting they be turned off and fixed but I forgot.

These are the lint errors returned with them off.

```
Messages
========

lifelines/estimation.py
  Line: 2
    pylint: unused-import / Unused BaseFitter imported from lifelines.fitters
  Line: 3
    pylint: unused-import / Unused WeibullFitter imported from lifelines.fitters.weibull_fitter
  Line: 4
    pylint: unused-import / Unused ExponentialFitter imported from lifelines.fitters.exponential_fitter
  Line: 5
    pylint: unused-import / Unused NelsonAalenFitter imported from lifelines.fitters.nelson_aalen_fitter
  Line: 6
    pylint: unused-import / Unused KaplanMeierFitter imported from lifelines.fitters.kaplan_meier_fitter
  Line: 7
    pylint: unused-import / Unused BreslowFlemingHarringtonFitter imported from lifelines.fitters.breslow_fleming_harrington_fitter
  Line: 10
    pylint: unused-import / Unused CoxPHFitter imported from lifelines.fitters.coxph_fitter
  Line: 11
    pylint: unused-import / Unused CoxTimeVaryingFitter imported from lifelines.fitters.cox_time_varying_fitter
  Line: 12
    pylint: unused-import / Unused AalenAdditiveFitter imported from lifelines.fitters.aalen_additive_fitter
  Line: 13
    pylint: unused-import / Unused AalenJohansenFitter imported from lifelines.fitters.aalen_johansen_fitter

lifelines/fitters/__init__.py
  Line: 20
    pylint: unused-variable / Unused variable 'estimate' (col 12)

lifelines/fitters/aalen_additive_fitter.py
  Line: 176
    pylint: unused-variable / Unused variable 'n' (col 8)
  Line: 300
    pylint: unused-variable / Unused variable 'n' (col 8)
  Line: 435
    pylint: unused-variable / Unused variable 'd' (col 11)

lifelines/fitters/aalen_johansen_fitter.py
  Line: 210
    pylint: unused-variable / Unused variable 'i' (col 12)

lifelines/fitters/cox_time_varying_fitter.py
  Line: 14
    pylint: unused-import / Unused solve imported from numpy.linalg
  Line: 19
    pylint: unused-import / Unused check_for_overlapping_intervals imported from lifelines.utils
  Line: 188
    pylint: unused-argument / Unused argument 'stop_times_events' (col 18)
  Line: 210
    pylint: unused-variable / Unused variable 'ei' (col 12)
  Line: 326
    pylint: unused-variable / Unused variable 'n' (col 8)

lifelines/fitters/coxph_fitter.py
  Line: 258
    pylint: unused-variable / Unused variable 'n' (col 8)
  Line: 611
    pylint: unused-argument / Unused argument 'T' (col 17)

lifelines/fitters/exponential_fitter.py
  Line: 48
    pylint: unused-argument / Unused argument 'entry' (col 8)

lifelines/fitters/weibull_fitter.py
  Line: 94
    pylint: unused-argument / Unused argument 'entry' (col 8)

lifelines/generate_datasets.py
  Line: 82
    pylint: unused-argument / Unused argument 't' (col 14)
    pylint: unused-argument / Unused argument 'alpha' (col 17)
  Line: 127
    pylint: unused-argument / Unused argument 'constant' (col 40)
  Line: 273
    pylint: unused-argument / Unused argument 'constant' (col 21)
    pylint: unused-argument / Unused argument 'independent' (col 37)
    pylint: unused-argument / Unused argument 'n_binary' (col 52)

lifelines/statistics.py
  Line: 331
    pylint: unused-argument / Unused argument 'test_result' (col 23)

lifelines/utils/__init__.py
  Line: 606
    pylint: unused-variable / Unused variable 'd' (col 7)
  Line: 1343
    pylint: unused-argument / Unused argument 'final_stop_time' (col 71)
    pylint: unused-variable / Unused variable 'construct_new_timeline' (col 4)



Check Information
=================
         Started: 2018-12-03 17:03:01.979813
        Finished: 2018-12-03 17:03:44.314070
      Time Taken: 42.33 seconds
       Formatter: grouped
        Profiles: .prospector.yaml, strictness_medium, strictness_high, strictness_veryhigh, no_doc_warnings, no_member_warnings, no_test_warnings
      Strictness: from profile
  Libraries Used:
       Tools Run: dodgy, mccabe, pep8, profile-validator, pyflakes, pylint, pyroma
  Messages Found: 34
```

Reply:
seeing all this information makes me so excited to fix them haha!